### ISSUE TITLE
Win 8.1 64 bit error recording

### ISSUE THREAD

**Issue by hejazb** [NapoleonWils0n/kodi-playercorefactory#67]
Hi Dear: Sir
I tried to install windows installer Ver 1.8, 2.0 and 2.2, If I install a Administrator, installation goes with no problem but when I run Kodi 16.1, my recordings turn off immediately, If I run kodi as Administrator some streams like CBS news on Cloud records but after showing the same error. If I run Kodi as a user with Admin privilege, it stops immediately, strangely if I run Kodi as Admin and try to record from Learning Zone site, it just kicks me out. I do not have problem using command line "bash rip-record video-url.txt.
I do appreciate if you can shed some light here and wish you the Best of Health.
Harry
d: line 72: cannot create temp file for here-document: Permission denied
d: line 76: cannot create temp file for here-document: Permission denied
/cygdrive/c/Users/Boby/git/kodi-playercorefactory/bash-scripts-windows/rip-recor
d: line 77: cannot create temp file for here-document: Permission denied
/cygdrive/c/Users/Boby/git/kodi-playercorefactory/bash-scripts-windows/rip-recor
d: line 78: cannot create temp file for here-document: Permission denied
/cygdrive/c/Users/Boby/git/kodi-playercorefactory/bash-scripts-windows/rip-recor
d: line 79: cannot create temp file for here-document: Permission denied

**Comment by NapoleonWils0n** [comment]
Hi Harry, good to hear from you mate
The issue is actually with the Windows installer and not the scripts

The issue is the installer doesnt add write permissions on the tmp folder in the cygwin folder
Version 1.8 and 2.0 use cygwin, version 2.2 doesnt use cywgin for bash

So to fix the issue with version with the tmp folder permission with version 1.8 and 2.0 of the installer,
heres what you need to do 

* You need to add write permission to this folder

```
C:\Program Files (x86)\Rip-Record\Cygwin\tmp
```

The Windows installer was written by another user called the t3rmin8tor,
who is a Windows certified engineer.

If you raise an issue about the tmp folder permission on the github page for the Windows installer below, the t3rmin8tor will be able to tell you how to fix the permission issue on the tmp folder

https://github.com/t3rmin8tor/kodi-player-core-factory-install

Also you can now change the recording destination for recording from the Desktop,
to any folder you like using a config file

* ripconfig - change recording destination video

https://www.youtube.com/watch?v=-AwmB5XHstM

* ripconfig file and read me on github

https://github.com/NapoleonWils0n/kodi-playercorefactory/tree/master/ripconfig

If you get stuck or have any other questions let me know

All the best mate

**Comment by hejazb** [comment]
Hi Dear: Friend Napoleon

Thank you so much for a very quick response, I will try to see if I can change the permission if, no then I might contact the contributor to this project.

I wonder if you have come up with an .APK installer for Android smartphones for recording web streams. I have a Asus Zenfone 2 4GB RAM 64GB ROM with intel Atom chip set (Quad processor), is it possible to record with such a device?

Again with success and a prosperous life


Harry

 

Gesendet: Dienstag, 28. Februar 2017 um 09:45 Uhr
Von: NapoleonWils0n <notifications@github.com>
An: NapoleonWils0n/kodi-playercorefactory <kodi-playercorefactory@noreply.github.com>
Cc: hejazb <hejazb@web.de>, Author <author@noreply.github.com>
Betreff: Re: [NapoleonWils0n/kodi-playercorefactory] Win 8.1 64 bit error recording (#67)


Hi Harry, good to hear from you mate
The issue is actually with the Windows installer and not the scripts

The issue is the installer doesnt add write permissions on the tmp folder in the cygwin folder
Version 1.8 and 2.0 use cygwin, version 2.2 doesnt use cywgin for bash

So to fix the issue with version with the tmp folder permission with version 1.8 and 2.0 of the installer,
heres what you need to do


	You need to add write permission to this folder


C:\Program Files (x86)\Rip-Record\Cygwin\tmp


The Windows installer was written by another user called the t3rmin8tor,
who is a Windows certified engineer.

If you raise an issue about the tmp folder permission on the github page for the Windows installer below, the t3rmin8tor will be able to tell you how to fix the permission issue on the tmp folder

https://github.com/t3rmin8tor/kodi-player-core-factory-install

Also you can now change the recording destination for recording from the Desktop,
to any folder you like using a config file


	ripconfig - change recording destination video


https://www.youtube.com/watch?v=-AwmB5XHstM


	ripconfig file and read me on github


https://github.com/NapoleonWils0n/kodi-playercorefactory/tree/master/ripconfig

If you get stuck or have any other questions let me know

All the best mate

**Comment by NapoleonWils0n** [comment]
Hi Harry
I just installed version 2.2 of the installer and had the same problem with the tmp folder

On version 2.2 of the installer the location of the tmp folder is:

```
C:\Program Files (x86)\Rip-Record\Git\tmp
```

you need to change the permissions on the tmp folder so normal users have write access

i right clicked on the C:\Program Files (x86)\Rip-Record\Git\tmp folder,
then selected the security tab

then under Group or user names i selected Users (Desktop-P350F4U\Users) and clicked the edit button

then in the next pop up in the Group or user names i selected users again,
and in the bottom panel selected full control and then clicked ok

then i rebooted
but after i rebooted i got an error when i ran the scripts saying command not found

im not a windows users, 
so would be best to raise an issue on the installer project page with the t3rminator

the issue is the tmp folder needs to have write permissions for normal users

re: Android

I havent got any further with Android,
so unless someone who knows Android can help out its a bit of a dead end

So at the moment its not possible to record on Android

**Comment by NapoleonWils0n** [comment]
Hi Harry 

I emailed the t3rminator ( guy who wrote the installer ) about the tmp folder issue with the installer

So if you raise an issue on the installer project page he should be able to resolve the issue for you

all the best

**Comment by hejazb** [comment]
I went to that folder shared it and then went to security property of the folder, gave myself all the rights that was available, after that it is working like a charm.

Thank you so much for your very precious help, it saved me a lot of time.

Again thank you and all the best


Harry

 

Gesendet: Dienstag, 28. Februar 2017 um 11:37 Uhr
Von: NapoleonWils0n <notifications@github.com>
An: NapoleonWils0n/kodi-playercorefactory <kodi-playercorefactory@noreply.github.com>
Cc: hejazb <hejazb@web.de>, Author <author@noreply.github.com>
Betreff: Re: [NapoleonWils0n/kodi-playercorefactory] Win 8.1 64 bit error recording (#67)


Hi Harry

I emailed the t3rminator ( guy who wrote the installer ) about the tmp folder issue with the installer

So if you raise an issue on the installer project page he should be able to resolve the issue for you

all the best
